src/Powercord/migrations/20220908-move-coremods-settings.js

```diff
@@ -0,0 +1,59 @@
+const { existsSync, renameSync } = require('fs');
+
+module.exports = () => {
+  // Check if coremod settings exist.
+
+  if (existsSync(`${__dirname}/../../../settings/pc-general.json`)) {
```

Comment:
Create variable with settings path so you don't have to repeat that

You could also make a helper function so you don't have to repeat as much. Or maybe have an object with all the paths and loop through that.